### PR TITLE
Edit performance part

### PR DESCRIPTION
**Pull request by LaureBro**

### CODE REVIEW

File: troubleshooting/bug_qualification/index.rst [hunk 1]
```diff
@@ -53,7 +53,7 @@ Tasks bugs are issues which occur when running the PIM jobs, like mass edit, imp
 Performances issues
 -------------------
 
-When the PIM is sloooowwwww, it's probably a performance issue.
+If you notice slowdowns: if it takes time to display pages, if imports or exports are running for ever, you are probably experimenting a performance issue.
```
Comment (rybus) [inline troubleshooting/bug_qualification/index.rst]: lots of "ifs" ;) When you experience slownesses, displaying pages and/or during import/exports.

**Comment by a2xchip** [comment]
`experimenting` - `experiencing`

**Comment by a2xchip** [comment]
:+1: @LaureBro

File: troubleshooting/bug_qualification/index.rst (hunk 1 above)
Comment (LaureBro) [inline troubleshooting/bug_qualification/index.rst]: fixed @rybus

**Comment by mickaelandrieu** [comment]
Thank you @LaureBro and @a2xchip / @rybus for reviews

**Pull request merged**